generator/index.ts

```diff
@@ -104,18 +109,28 @@ const clientAPIInfo: string = execSync(
   "./node_modules/.bin/documentation build api/client.js --document-exported --shallow --markdown-toc false -f md",
   { encoding: "utf8" }
 );
+
+const facadeList = {};
```

Comment:
I'm surprised this untyped object doesn't produce a type error. That might be to do with the typescript settings in this project.